misc/formatter/modules/formatter-cli/src/test/java/org/ballerinalang/formatter/cli/FormatCmdTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.formatter.cli;
+
+import org.ballerinalang.tool.BLauncherException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Format CLI tool test suit for testing tool's exceptions.
+ */
+public class FormatCmdTest {
+    private static final Path RES_DIR = Paths.get("src/test/resources/").toAbsolutePath();
```

Comment:
Will this work in windows ? Using `resolve` function would help. `Paths.get("src").resolve("test").resolve("resources").toAbsolutePath()

Reply:
Check for other similar places

Reply:
Done